source/components/automation.markdown

```diff
@@ -246,3 +246,16 @@ automation:
 If you want to specify multiple services to be called or include a delay, have a look at the
 [script component](/components/script.html). If you want to describe how certain entities should look,
 check out the [scene component](/components/scene.html).
+
+Troubleshooting Tip: You can verify that your automation rules are being initialized correctly by watching both the realtime logs and also the `logbook`.  The realtime logs will show the rules being initialized (once for each trigger)
+```bash
+INFO (MainThread) [homeassistant.components.automation] Initialized rule Rainy Day
+INFO (MainThread) [homeassistant.components.automation] Initialized rule Rainy Day
+INFO (MainThread) [homeassistant.components.automation] Initialized rule Rainy Day
+INFO (MainThread) [homeassistant.components.automation] Initialized rule Rain is over
+```
+
+The `LogBook` component will show a line entry when an automoation is triggered.  You can look at the previous entry to determine which trigger in the rule triggered the event.</br>
+
+![Sample](http://i.imgur.com/2s1ukCb.png)
```

Comment:
Please do not try to sneak in imgur hosted images again. All used images have to be part of this repository and served from the home-assistant domain over SSL.